.github/workflows/integration_tests.yml

```diff
@@ -18,7 +18,7 @@ on:
         required: true
       apis:
         description: 'CSV of apis to build and test'
-        default: 'analytics,auth,crashlytics,database,dynamic_links,firestore,functions,installations,messaging,remote_config,storage'
+        default: 'analytics,app_check,auth,crashlytics,database,dynamic_links,firestore,functions,installations,messaging,remote_config,storage'
```

Comment:
What tests does this run for app_check?

Reply:
Currently basically nothing, the test suite is empty.  I'll be adding those in soon.